test/lib/test_utils.hh

```diff
@@ -113,10 +113,16 @@ extern std::mutex boost_logger_mutex;
 
 }
 
+namespace internal {
+
+template <typename T, typename Char = char>
+concept formattable = fmt::is_formattable<std::remove_reference_t<T>, Char>::value;
+
+}
+
 namespace std {
 
-template <typename T>
-requires fmt::is_formattable<T>::value
+template <::internal::formattable T>
 std::ostream& boost_test_print_type(std::ostream& os, const T& p) {
```

Comment:
I'm surprised this is necessary. If the compiler has to resolve boost_test_print_type(foo_t&) then T should resolve to foo, not foo&. Anyway it's not worth digging into.

Reply:
good point. now i doubt it, and i don't think it's necessary after a second thought. will drop it in a follow-up change.